Documentation/DSCResources/VMHostVss/VMHostVssSecurity.md

```diff
@@ -0,0 +1,72 @@
+# VMHostVssSecurity
+
+## Parameters
+
+| Parameter | Attribute | DataType | Description | Allowed Values |
+| --- | --- | --- | --- | --- |
+| **Server** | Key | string | Name of the Server we are trying to connect to. The Server can be a vCenter or ESXi. ||
+| **Name** | Key | string | Name of the VMHost to configure. ||
+| **Credential** | Mandatory | PSCredential | Credentials needed for connection to the specified Server. ||
+| **Ensure** | Mandatory | Ensure | Value indicating if the VSS should be Present or Absent. | Present, Absent |
+| **VssName** | Mandatory | string | The name of the VSS. ||
+| **AllowPromiscuous** | Optional | boolean | The flag to indicate whether or not all traffic is seen on the port. | $True,$False |
```

Comment:
I think the values $True and $False here are redundant.
The same comment for the two properties below.

Reply:
Removed